index.html

```diff
@@ -14,18 +14,62 @@
   <link rel="stylesheet" href="https://cdn.rawgit.com/konpa/devicon/df6431e323547add1b4cf45992913f15286456d3/devicon.min.css">
   <link href="https://fonts.googleapis.com/css?family=Gloria+Hallelujah|Varela+Round" rel="stylesheet">
   <link rel="stylesheet" type="text/css" href="css/style.css" />
-  <title>Nonprofit Product Landing Page Design - Lunatic Apps</title>
+  <!-- Font -->
+  <link href="https://fonts.googleapis.com/css?family=Muli:400,700" rel="stylesheet">
+  <!-- Animate CSS -->
+  <link rel="stylesheet" href="css/animate.css">
 
 </head>
   <body>
-
+    <!-- START SECTION FEATURE#13, BUILT BY MARCIN WICHA-->
```

Comment:
Your code is looks good. You start well, I want to suggest if we write comments with names it'll more understandable for debugging code faster. Use uppercase comment when working in big sections. Here is my example:

 <!-- START ABOUT FEAT#13, BUILT BY MARCIN WICHA-->
...........
...........
<!-- END ABOUT FEAT#13, BUILT BY MARCIN WICHA-->